ReactiveCocoa/Swift/Property.swift

```diff
@@ -501,13 +504,15 @@ public final class Property<Value>: PropertyProtocol {
 				fatalError("Receive unexpected error from a producer of `NoError` type: \(error)")
 			}
 		}
+
+		// Verify that an initial is sent. This is friendlier than deadlocking
+		// in the event that one isn't.
 		guard value != nil else {
```

Comment:
Then you would have created a new constant `value` to shadow the original `value`.